django_webapps/scripts/db_population/parsers/student_parsers.py

```diff
@@ -0,0 +1,50 @@
+import re
+
+from api.models.auxil.constants import STUDENT_AGE_RANGES
+from api.models.choices.status.project import StudentProjectStatus
+from api.models.choices.status.situational import StudentSituationalStatus
+
+RAW_PROJECT_STATUS_TO_PARSED: dict[
+    str, tuple[StudentSituationalStatus | None, StudentProjectStatus]
+] = {
+    "группа расформирована": (None, StudentProjectStatus.FINISHED),
+    "не ходит на занятия": (StudentSituationalStatus.NOT_ATTENDING, StudentProjectStatus.STUDYING),
+    "идут занятия": (None, StudentProjectStatus.STUDYING),
+    "нет ответа": (StudentSituationalStatus.NO_RESPONSE, StudentProjectStatus.NO_GROUP_YET),
+    "перестал отвечать": (StudentSituationalStatus.NO_RESPONSE, StudentProjectStatus.NO_GROUP_YET),
```

Comment:
[Confirmed with the owners](https://t.me/c/1808422267/3115) that both cases mean that project status is "no group yet" 👍🏻